test/connectivity/NetworkController/ScheduledTask.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+
+    class ScheduledTask
```

Comment:
Can we re-use PeriodicTask to have an overload that runs n number of times?

Reply:
actually I like this implementation as it is simple and run synchronously.

Reply:
Its async...
but I am fine with it.. not much code.